src/network/network_details_templates.js

```
@@ -377,12 +377,17 @@ templates._request_body = function(req, do_raw)
 templates._response_body = function(resp, do_raw, is_last_response)
 {
   var ret = [];
-
   var classname = "";
-  if ((resp.saw_responsefinished && resp.no_used_mimetype) ||
-      !resp.responsebody && resp.is_unloaded)
+
+  var should_track_content = (
```

I thought of semicolon insertion, I don't know the rules well enough there. With a return statement this would be a very silent error, so maybe reason enough to keep so there's no doubt when reading it?